content/en/catalyst/fund9/challenges/grow-africa-grow-cardano/campaign-brief.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Grow Africa, Grow Cardano"
+linkTitle: "Grow Africa, Grow Cardano"
+type: docs
+hide_feedback: true
+description: "How do we continue to seed and grow Cardano in Africa in 2022?"
+---
+<img src="https://cardano.ideascale.com/community-library/accounts/93/936143/Public/10-Grow-Africa-Grow-Cardano-1daa76.png" style="width:500px;height500px">
+
+Fund Size: **500,000 USD paid in Ada**
+
+[Campaign page](https://cardano.ideascale.com/c/idea/396613) and [browse ideas]().
+
+## Challenge Brief
+It's no secret that Cardano is recognized as a blockchain with a strong focus on Africa, and formally stated as such in Cardano's strategic goals. However, as other blockchain protocols recognize the potential of crypto in Africa, without concrete sustained action, Cardano risks losing a potential $1 trillion in market share of the African market to other blockchains within the next 3 years.
+
+- Africa's crypto market has grown by $105 billion in the last year (2020-2021). Africa is the next crypto frontier- World Economic Forum
+- Alameda Research, Coinbase Ventures, Cadenza, CMT Digital, Distributed Global, GSR, Third Prime and Avon Ventures: [African Crypto Exchange VALR Raises $50M in Series B Led by Pantera Capital](https://www.coindesk.com/business/2022/03/01/african-crypto-exchange-valr-raises-50m-in-series-b-led-by-pantera-capital/)
+
+In Catalyst Fund7 the combined Africa funded projects amount was almost negligible compared to those from [western and developed economies](https://twitter.com/danny_cryptofay/status/1496078645319933954?s=20&t=Ohr06EIZW05vz_B4Fjk_JA)
+
+This needs to be reversed through intentional action. Otherwise smaller and smaller portions of Catalyst funding will continue to be allocated to projects focused on African and other emerging economies due to endemic barriers.
+
+"Having a funding round that focuses on Africa is a win, win, win scenario. Africa wins by getting some backing. Commercial projects that are already building up in Africa win, because they get the signal of support from our community. Cardano holders win when there is an encouraging environment for mass adoption anywhere. Focusing on Africa is a strategy that has often been promoted and is openly pursued by IOG and Charles Hoskinson. The ROI is likely to be high because community sponsored projects are better run in young and developing economies." Greg Bell; (originator of the Grow Cardano series)
+
+### Rationale for budget
+
+Let's be frank here. We need much more than 250k to continue seeding Cardano in Africa. We are asking for this amount because of Catalyst voting patterns; in that $250k is better than $0 and Wada and Partners can still take advantage of other challenge categories to help expand Cardano in Africa.
+
+### Why is it important?
```

Comment:
The 'Why is it important' should be written, 'We need to look at the fundamentals that are preventing Africa from being a Cardano adoption leader, rather than a perpetual follower.'